spanner/tests/system/test_system.py

```diff
@@ -728,153 +698,150 @@ def _row_data(max_index):
                 'test-%09d@example.com' % (index,),
             ]
 
-    def _set_up_table(self, row_count, db=None):
-        if db is None:
-            db = self._db
+    def _set_up_table(self, row_count, database=None):
+        if database is None:
+            database = self._db
             retry = RetryInstanceState(_has_all_ddl)
-            retry(db.reload)()
-
-        session = db.session()
-        session.create()
-        self.to_delete.append(session)
+            retry(database.reload)()
 
         def _unit_of_work(transaction, test):
             transaction.delete(test.TABLE, test.ALL)
             transaction.insert(
                 test.TABLE, test.COLUMNS, test._row_data(row_count))
 
-        committed = session.run_in_transaction(_unit_of_work, test=self)
+        committed = database.run_in_transaction(_unit_of_work, test=self)
 
-        return session, committed
+        return committed
 
     def test_read_with_single_keys_index(self):
         row_count = 10
         columns = self.COLUMNS[1], self.COLUMNS[2]
-        session, committed = self._set_up_table(row_count)
-        self.to_delete.append(session)
+        committed = self._set_up_table(row_count)
+
         expected = [[row[1], row[2]] for row in self._row_data(row_count)]
         row = 5
         keyset = [[expected[row][0], expected[row][1]]]
-        results_iter = session.read(self.TABLE,
-                                    columns,
-                                    KeySet(keys=keyset),
-                                    index='name'
-        )
-        rows = list(results_iter)
-        self.assertEqual(rows, [expected[row]])
+        with self._db.snapshot() as snapshot:            
+            results_iter = snapshot.read(self.TABLE,
```

Comment:
since you're here, fix this hanging indent (line break at the first `(` and just indent once).

Reply:
changed this and other ones like it to 
```
function(
  arg1,
  arg2,
)
```